os/arch/arm/src/imxrt/Make.defs

```diff
@@ -168,6 +168,8 @@ endif
 
 ifeq ($(CONFIG_IMXRT_ENET),y)
 CHIP_CSRCS += imxrt_enet.c
+else
```

Comment:
Would it make sense to add a CONFIG_HAVE_NET check here for building imxrt_net.c, just in case a developer wishes to disable network stack? Please ignore remark if not necessary.